app/views/content/life-as-a-teacher/age-groups-and-specialisms/secondary/_article.html.erb

```diff
@@ -0,0 +1,89 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach <%= @front_matter["subject"] %></h2>
+    <div class="inset">
+      <p>Becoming a secondary school teacher is a great career choice if you want to make a difference in young people's lives. As students progress through school, you will directly influence their future prospects through the grades you help them achieve and the values you share with them.</p>
+      <p>As a specialist, you'll inspire students to develop a passion for your subject. You'll give them the skills and knowledge needed to tackle increasingly complex content and motivate some of them to pursue your subject beyond school.</p>
+      <!-- Why teach: benefits -->
+      <%= render 'content/shared/teaching/benefits' %>
+    </div>
+      <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_art_richard' %>
+    </section>
+    <h2 class="heading--box-blue">Key stages</h2>
+    <p>Pupils start secondary school in England when they’re 11. Throughout secondary school, pupils will be taught by subject specialists and will gain increasingly greater independence over their own learning.</p>
+    <p> There are 3 stages of secondary education.</p>
```

Comment:
Maybe worth adding a little bit here about what key stages are. There's a line on gov which summarises this but obvs needs changing to fit the GIT tone of voice:

'The national curriculum is organised into blocks of years called ‘key stages’ (KS). At the end of each key stage, the teacher will formally assess your child’s performance.'